add CODEOWNERS

thraxil opened the pull request.

thraxil committed:
add CODEOWNERS

bryanlandia commented on .github/CODEOWNERS:
```diff
@@ -0,0 +1,3 @@
+# see: https://appsembler.atlassian.net/wiki/spaces/ED/pages/2227503161/CODEOWNERS
+
+* @bryanlandia
```
```suggestion
* @bryanlandia @briandant 
```

bryanlandia reviewed (changes requested):
Suggested adding @briandant

thraxil committed:
Update .github/CODEOWNERS

Co-authored-by: Bryan Wilson <bryanlandia@gmail.com>

briandant commented:
@thraxil I saw a comment somewhere that for SOC 2 there should be only one code owner, is that right? https://appsembler.atlassian.net/wiki/spaces/ED/pages/2227503161/CODEOWNERS

briandant committed:
we need to have one primary codeowner

see 

> Subsequent lines in the file can specify multiple users or teams as the default reviewers. That is fine and teams are encouraged to set that up in whatever way makes the most sense for them. We just need that first line to designate a single individual.

in https://appsembler.atlassian.net/wiki/spaces/ED/pages/2227503161/CODEOWNERS
